cmd/wallet-web/src/pages/Dashboard.vue

```diff
@@ -114,8 +114,16 @@ export default {
   },
   methods: {
     ...mapGetters('agent', { getAgentInstance: 'getInstance' }),
-    ...mapGetters(['getCurrentUser', 'getAgentOpts']),
+    ...mapGetters(['getCurrentUser', 'getAgentOpts', 'getStaticAssetsConfigData']),
     ...mapActions(['updateProcessedCredentials']),
+    // Get credentialDisplayData based on docker configuration
+    getCredentialDisplayDataConfig: async function () {
+      const credentialConfigData = await this.getStaticAssetsConfigData();
+      if (credentialConfigData) {
+        return (this.credentialDisplayData = credentialConfigData);
+      }
+      return (this.credentialDisplayData = require('@/config/credentialDisplayData.js').default);
```

Comment:
this switching between local vs remote static config should happen in centralized place like vuex store. Doesn't make sense to load it again and again each time this template refreshes.

Reply:
I removed the switching from here, realized should have done in the option.js file itself, Option.js file also reside in the same common folder as vuex store.